deployment/docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+services:
+
+  nestjs:
+    container_name: nestjs
+    build:
+      context: ../
+      dockerfile: ./deployment/Dockerfile
+    depends_on:
+      - postgres
+    env_file:
+      - .env.dev
```

Comment:
Be very careful with sharing whole env variables with all containers.
There might be some secret env variable that we don't want to end-up in all containers, like front-end.

Reply:
So we need to create different env files for each service ?

Reply:
Sorry pal, merging all .env files into a single file was my idea 😦